### PR TITLE
Add support for big-image

### LINKED ISSUE #34: big-image and media::big-image column heading for itext tag is not available in XLSForm
There is a "big-image" itext value tag type that is not supported by XLSForm.

Usage section below is copied from this page: https://opendatakit.org/help/form-design/guidelines/

---

Itext value tag types

The possible choices for the form attribute of the itext <value> tag are:

```
"long",
"short",
"image",
"audio",
"video", and
"big-image".
```

By default, itext "image" values are not clickable. However, if you also include a "big-image", the image displayed by "image" will be clickable and will display a pannable, zoomable view of the file specified by "big-image". You can return to your form after opening a "big-image" by hitting the phone's "back" button. Specifying "big-image" alone has no effect, you must always include "image".

### PR DESCRIPTION
**Pull request by lognaturel**
Closes #34

#### Why is this the best possible solution? Were any other approaches considered?
I tried to follow the implementation for other media types exactly. I didn't consider any alternatives.

I considered not adding `big-image` to all of the translation tests but I decided it would feel more consistent to do so.

#### What are the regression risks?
This is an additive change that doesn't touch much existing logic so regression risks are low. There's some possibility of regression with other media types but they're well tested.

#### Does this change require updates to documentation? If so, please file an issue [here](https://github.com/XLSForm/xlsform.github.io) and include the link below.

Yes, https://github.com/XLSForm/xlsform.github.io/issues/234

#### Before submitting this PR, please make sure you have:
- [x] included test cases for core behavior and edge cases in `tests`
- [x] run `nosetests` and verified all tests pass
- [x] run `black pyxform tests` to format code
- [x] verified that any code or assets from external sources are properly credited in comments

### CODE REVIEW

File: pyxform/survey_element.py [hunk 1]
```diff
@@ -428,10 +428,15 @@ def xml_label_and_hint(self) -> "List[DetachableElement]":
         msg = "The survey element named '%s' " "has no label or hint." % self.name
         if len(result) == 0:
             raise PyXFormError(msg)
+
         # Guidance hint alone is not OK since they may be hidden by default.
         if not any((self.label, self.media, self.hint)) and self.guidance_hint:
             raise PyXFormError(msg)
 
+        # big-image must combine with image
+        if not ("image" in self.media) and "big-image" in self.media:
```
Comment (lindsay-stevens) [inline pyxform/survey_element.py:437]: Suggest `if "image" not in self.media and "big-image" in self.media`

File: pyxform/survey_element.py [hunk 2]
```diff
@@ -428,10 +428,15 @@ def xml_label_and_hint(self) -> "List[DetachableElement]":
...
+            raise PyXFormError("You must specify an image in order to use big-image.")
```
Comment (lindsay-stevens) [inline pyxform/survey_element.py:438]: Suggest `raise PyXFormError(f"In order to use big-image, you must specify an image for the survey element named {self.name}`

**Review by lindsay-stevens — APPROVED** [review]
LGTM. 2 minor suggestions around validation/message.

**Comment by lindsay-stevens** [comment]
One other thought, is it fine that the translation test `test_missing_translation__one_lang_all_cols__warn__default` passes, seeing as the question type is `note` and there's a `big-image` specified?

**Comment by lognaturel** [comment]
> the question type is note and there's a big-image specified

Yes, this is fine and good! `big-image` can be used to provide a bigger image either for a question label or for a select choice. A `note` is a read-only text question and can have the same label types as other question types.

**Comment by lognaturel** [comment]
For posterity/anyone watching who may have opinions, I want to acknowledge that this is the first column with a - in its name, all others use underscore.

I went with it because it matches the underlying XForm spec and is more of a type/variant than the other column headers.
